src/lolex-src.js

```diff
@@ -626,6 +626,10 @@ function withGlobal(_global) {
             var nanosTotal = nanos + remainder;
             var tickTo = clock.now + ms;
 
+            if (msFloat < 0) {
+                throw new TypeError("Only positive ticks are supported");
```

Comment:
The error message is a bit misleading. Are zero ticks allowed?
If not, shouldn't the conditional be `msFloat <= 0`?

Reply:
I'm also fine with ``.tick` ms value must be called with a value >= 0` or something like that.

Reply:
Changing to `Negative ticks are not supported`